app.rb

```diff
@@ -0,0 +1,182 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './book'
+
+# rubocop:disable Metrics/ClassLength
```

Comment:
- [ ] You have done an excellent job and we greatly appreciate your hard work. The UI ran without any errors. However, the use of the rubocop:disabler is not allowed in this project. Please adjust accordingly and resubmit. Thank you for your consideration. 

> Suggestion: I think it would be good idea to ask from main.rb to give some spare room to some of your app.rb methods such `run` method and `library_menu` methods. Thank you for your understanding.

_Reach me out if you need any help 🔢_

Reply:
I am having trouble trying to call the `run` and `library_menu`  from main.rb, can I get some more help on that?

Reply:
Hi @kaizipaul, Yes of course.

Reply:
Could you reach me out via slack?